test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -65,10 +68,13 @@
 	private static final String FILE_SEPARATOR = System.getProperty("file.separator");
 	private static final String USER_DIR = System.getProperty("user.dir");
 	private static final String OS_NAME_PROPERTY = System.getProperty("os.name");
+	private static final String VMARGS_JAR = "vmargs_SE80.jar";
 	private static final String XJIT = "-Xjit";
 	private static final String XINT = "-Xint";
 	private static final String XPROD = "-Xprod";
 	/* set to true if the implicit VM arguments take priority over IBM_JAVA_OPTIONS and JAVA_TOOL_OPTIONS */
+	private static final String HELLO_WORLD = "#!/bin/sh\n"
```

Comment:
Likely clearer to name this `HELLO_WORLD_SCRIPT`.  If changing this anyway, then I merge the two lines into a single one.

Reply:
Okay.